### ISSUE TITLE
NVDA mute after open a program by the start menu.

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#6285]
I have an intermittent error sometimes when opening a program by just start menu by pressing the enter on the program in the start menu NVDA is muted.
For NVDA speak again you need to open the Start menu again.
The error does not occur every time, and so it does not happen again is to restart NVDA.
Use NVDA with add-ons disabled does not work.
I'm using NVDA next 13513, and the windows 10.0.14393.51.
The error reported in the log NVDA is:
Input: kb (laptop): upArrow
IO - speech.speak (10:43:46):
Speaking [u "COMError: (-2147220991," An event \ n \ p xe3o xf4de call 
[nvda-old.log.txt](https://github.com/nvaccess/nvda/files/428439/nvda-old.log.txt)
subscribers' (None, None, None, 0, None)) \ r "]
The complete log is also attached.

**Comment by Brian1Gaff** [comment]
Yes and sometimes it does this after an nvda update as well. The other day I 
thought these weirdnesses had been fixed, but I spoke too soon. Oftnen no 
error is visible in the log either, its as if its still speaking but into 
somewhere else. Could this be an issue with the way Windows handles sound as 
I've never had this issue in xp.
 Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal email to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
----- Original Message -----

**Comment by ehollig** [comment]
@fernando-jose-silva, is this still an issue? I also recall that there was possibly a duplicate of this issue somewhere.

**Comment by fernando-jose-silva** [comment]
Much has changed from there to now.
Using the updated versions of windows 10 17133 and nvda next 15040 I no longer have this problem.
Thank you very much.

**Comment by ehollig** [comment]
Thank you @fernando-jose-silva. Closing as worksforme from https://github.com/nvaccess/nvda/issues/6285#issuecomment-382761475
